src/umpire/resource/CMakeLists.txt

```diff
@@ -11,12 +11,17 @@ set (umpire_resource_headers
   MemoryResource.hpp
   MemoryResourceFactory.hpp
   MemoryResourceRegistry.hpp
-  MemoryResourceTypes.hpp)
+  MemoryResourceTypes.hpp
+  NullMemoryResource.hpp
+  NullMemoryResourceFactory.hpp
+)
 
 set (umpire_resource_sources
   HostResourceFactory.cpp
   MemoryResource.cpp
   MemoryResourceRegistry.cpp
+  NullMemoryResource.cpp
```

Comment:
Should we guard this with ENABLE_ZERO_BYTE

Reply:
I'm not sure...

Reply:
We agree not to guard this.